test/ventilation-test.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * ventilation-test.c
+ *
+ * test the function contain in ventilation.c
+ */
+
+#include "application/ventilation.h"
+#include "stubs/systemInfo_stub.h"
+#include "stubs/valve_stub.h"
+#include "testUtil.h"
+
+
+void reset_to_inspiration_start();
+
+void startVentilation_openInspiratoryValve() {
+  int* times = (int*) malloc(2*sizeof(int)); 
+  times[0] = 130;
+  times[1] = 290;
+  set_current_time(times);
+
+  close_inspiratory_valve();
+  
+  start_inspiration();
+
+  int inspiratory_valve_status = get_inspiratory_valve_status();
+  assertTrue(inspiratory_valve_status == VALVE_OPEN);
+
+  close_inspiratory_valve();
+  reset_to_inspiration_start();
+
+  start_inspiration();
+
+  inspiratory_valve_status = get_inspiratory_valve_status();
+  assertTrue(inspiratory_valve_status == VALVE_OPEN);
```

Comment:
So this will just print the assertion result right? It doesn't tell me if the test actually passes or fails? How do you communicate that back to the test framework so it can aggregate passes/fails for you. 

If we have 100s of tests, I don't want to have to dig through the test logs to find if any tests passed/failed. I should be able to see it from a single line at the end of the test log: `2 out of 2 tests PASSED` or something like that.